src/partials/_button.module.scss

```diff
@@ -1,77 +1,53 @@
-@use "./variables" as *;
-@use "./breakpoints" as *;
+@use './variables' as *;
+@use './breakpoints' as *;
 
-
-%default-style{
-  @include modal-button-text;
-  display:flex;
+%default-style {
+  display: flex;
   justify-content: center;
   align-items: center;
-  background-color: $purple-primary;
-  height: 40px;
-  width: 295px; //default width set to mobile modal button size
-  border: none;
   border-radius: 24px;
-  padding: 0;
+  border: none;
+  padding: 16px;
   margin: 0;
-  color: $white;
   user-select: none;
   cursor: pointer;
+  font-weight: bold;
 
-  &:hover{
-    background-color: $purple-secondary;
+  &.sm {
+    font-size: 13px;
   }
-}
-
-// Buttons for Modals
-.default{
-  @extend %default-style;
 
-//breakpoint changing button size from 295px to 416px
-  @include lg{
-    width: 416px;
+  &.lg {
+    font-size: 15px;
   }
```

Comment:
individual components can work without defining breakpoint sizes, let the parent container handle this